pandas/core/arrays/arrow/array.py

```diff
@@ -1320,6 +1321,7 @@ def to_numpy(
         copy: bool = False,
         na_value: object = lib.no_default,
     ) -> np.ndarray:
+        original_na_value = na_value
```

Comment:
Just wondering, shouldn't this case be handled in `to_numpy_dtype_inference`?

Reply:
The scope of the function is if no dtype is given, we can extend this, but I don’t know if that’s a good idea